map/map_tf_generator/include/map_tf_generator/uniform_random.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef MAP_TF_GENERATOR__UNIFORM_RANDOM_HPP_
+#define MAP_TF_GENERATOR__UNIFORM_RANDOM_HPP_
+
+#include <random>
+#include <vector>
+
+std::vector<int> UniformRandom(const int max_excluded, const int n)
```

Comment:
@kenji-miyake  How about here? Do I need to replace these int to size_t?

Reply:
If it's only for indices, I think it would be better.

Reply:
If you want to use this for `int` and `size_t`, you can consider using make it a template.

Reply:
Resolved in https://github.com/autowarefoundation/autoware.universe/pull/890/commits/0a2a601ac7906ad6d444a0c56fc37221ec11b3dd